client/js/table-model.js

```diff
@@ -1,17 +1,17 @@
 class TableModel{
-	constructor(numCols=5,numRows=7){
-		this.numCols = numCols;
-		this.numRows = numRows;
-		this.data = {};
-	}
-	_getCellId(location){
-		return `${location.col}:${location.row}`;
-	}
-	getValue(location){
-		return this.data[this._getCellId(location)];
-	}
-	setValue(location,value){
-		this.data[this._getCellId(location)] = value;
-	}
+  constructor(numCols=5,numRows=7){
+    this.numCols = numCols;
+    this.numRows = numRows;
+    this.data = {};
+  }
```

Comment:
Please use one blank line to separate methods in a class. This makes it easier to scan the whole file and see functions as chunks of code.